schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -809,8 +809,11 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/netAppAccounts" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/capacityPools" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/volumes" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/snapshots" }
-]
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-08-15/Microsoft.NetApp.json#/resourceDefinitions/snapshots" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2015-08-31-preview/Microsoft.ManagedIdentity.json#/resourceDefinitions/UserAssignedIdentities" },
```

Comment:
>UserAssignedIdentities [](start = 144, length = 22)

please use consistent casing as the schema definition. The first letter "u" should be lower case.